cumulusci.yml

```diff
@@ -451,67 +438,46 @@ flows:
             4:
                 task: deploy_pre
 
-    dev_org:
-        description: Set up an org as a development environment for unmanaged metadata
+    config_unmanaged:
         steps:
             1:
-                flow: beta_dependencies
-            2:
-                flow: deploy_unmanaged
-            3:
-                flow: config_dev
-            4:
                 task: enable_course_connections
-            5:
+            2:
                 task: deploy_trial_config
-            6:
+            3:
                 task: deploy_k12_kit_settings
-            7:
+            4:
                 task: execute_install_apex
-            8:
-                task: snapshot_changes
+    
+    config_dev:
+        steps:
+            3:
+                flow: config_unmanaged
+
+    config_qa:
+        steps:
+            3:
+                flow: config_unmanaged
 
     dev_org_namespaced:
         description: Set up a namespaced scratch org as a development environment for unmanaged metadata
         steps:
             1:
-                flow: beta_dependencies
+                flow: dependencies
             2:
                 flow: deploy_unmanaged
             3:
                 flow: config_dev
             4:
-                task: enable_course_connections
-                options:
-                    namespaced: True
-            5:
-                task: deploy_trial_config
-                options:
-                    namespaced_org: True
-            6:
-                task: deploy_k12_kit_settings
-            7:
-                task: execute_install_apex
+                flow: config_unmanaged
             8:
                 task: snapshot_changes
 
-    install_beta:
-        description: Install and configure the latest beta version
-        steps:
-            1:
-                flow: beta_dependencies
-            2:
-                task: install_managed_beta
-            3:
-                flow: config_managed
-            4:
-                task: snapshot_changes
-
     net_new_org:
         description: Simulates an K12 installation for a new customer. Installs the latest beta and dependencies.
         steps:
             1:
-                flow: beta_dependencies
+                flow: dependencies
```

Comment:
This might not be right depending on how the flow is used. The description says "Installs the latest beta and dependencies", but we would never deliver betas to a customer.

Reply:
This was there for QE's to test an upcoming new org experience before we release I believe.

Reply:
Hmm, like `regression_org` but without the update mechanism? Then this change is probably wrong and I'll revert it.

Reply:
I'm noticing that there remains a change for 'dependencies' here. There is something worth noting in that beta_dependencies captures the experience of a customer on a new org with the new version of EDA as well. My understanding is that this is important to QE. I do not see the change reverted. @jofsky @davidmreed

Reply:
@MatthewBlanski I added the `include_beta: True` option to maintain the existing behavior. The `beta_dependencies` flow is deprecated.

Reply:
@MatthewBlanski Do you have any concerns about the revised PR?

Reply:
This resolves my concerns.